src/bot.ts

```diff
@@ -500,9 +500,9 @@ export class DiscordBot {
             throw Error("Room(s) not found.");
         }
         const entry = entries[0];
-        const guild = client.guilds.get(entry.remote.get("discord_guild"));
+        const guild = client.guilds.get(entry.remote!.get("discord_guild") as string);
```

Comment:
why are we assuming `entry.remote` is defined? shouldn't we check for this somehow?

Same goes for all the other additions of `!`